ibc_data.schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": [
+    "chain-1",
+    "chain-2",
+    "channels"
+  ],
+  "properties": {
+    "chain-1": {
+      "type": "object",
+      "items": {
+        "$refs": "#/$defs/chain_info"
+      }
+    },
+    "chain-2": {
+      "type": "object",
+      "items": {
+        "$refs": "#/$defs/chain_info"
+      }
+    },
+    "channels": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "required": [
+            "chain-1",
+            "chain-2",
+            "ordering",
+            "version"
+          ],
+          "properties": {
+            "chain-1": {
+              "type": "object",
+              "items": {
+                "$refs": "#/$defs/channel_info"
+              }
+            },
+            "chain-2": {
+              "type": "object",
+              "items": {
+                "$refs": "#/$defs/channel_info"
+              }
+            },
+            "ordering": {
+              "enum": [
+                "ordered",
+                "unordered"
+              ],
+              "description": "Determines if packets from a sending module must be ORDERED or UNORDERED."
+            },
+            "version": {
+              "type": "string",
+              "description": "IBC Version"
+            },
+            "description" : {
+              "type":"string",
+              "description": "Human readable description describing the channel"
+            },
+            "tags": {
+              "type": "object",
+              "description": "Human readable key:value pair that describes what the path is servicing. E.g. 'dex': 'osmosis'",
+              "properties": {
+                "status": {
+                  "enum": [
+                    "live",
+                    "upcoming",
+                    "killed"
+                  ]
+                },
+                "preferred": {
+                  "type": "boolean"
+                },
+                "dex": {
+                  "type": "string"
```

Comment:
Neat!

I'm wondering if we might need to adapt in the future by adding new "properties" types to cover non-dex use-cases (such as interchain accounts, which have their own channels).

Reply:
Thanks for your sign-off here @adizere. `properties` tag added.